feat(cxl-ui): dashboard roadmap block

HenerHoop opened the pull request:
https://app.clickup.com/t/86aydn6th

HenerHoop commented:
Task linked: [CU-86aydn6th Roadmap Block](https://app.clickup.com/t/86aydn6th)

pawelkmpt reviewed (commented):
@HenerHoop I'm adding visual glitches here, but I'm not sure if they need adjustment in aybolit or WPS.

**1. Too big whitespace** 
Figma seems to have it smaller. It just feels empty anyway.

![Screenshot 2023-11-02 at 10 10 22](https://github.com/conversionxl/aybolit/assets/5165721/4b22b625-6ec5-4311-b65c-706cad712bf8)

---

**2. Shadow behind the content** 

![Screenshot 2023-11-02 at 10 11 04](https://github.com/conversionxl/aybolit/assets/5165721/a16b8cb7-5f3c-4364-895f-121e5e08199e)

It probably depends on some other branch. If so, please point me out  to it.

---

**3. Menu looks different**

This looks like a side effect of changes. I'll make sure I compiled everything correctly but wanted to highlight it anyway, maybe you see the same.

![Screenshot 2023-11-02 at 10 11 22](https://github.com/conversionxl/aybolit/assets/5165721/8884ab31-efe1-4efb-808b-1459aafaa9b4)

pawelkmpt reviewed (changes requested):
Few visual things. Check comments above.

HenerHoop commented:
> @HenerHoop I'm adding visual glitches here, but I'm not sure if they need adjustment in aybolit or WPS.
> 
> **1. Too big whitespace** Figma seems to have it smaller. It just feels empty anyway.
> 
> ![Screenshot 2023-11-02 at 10 10 22](https://user-images.githubusercontent.com/5165721/279928395-4b22b625-6ec5-4311-b65c-706cad712bf8.png)
> 
> **2. Shadow behind the content**
> 
> ![Screenshot 2023-11-02 at 10 11 04](https://user-images.githubusercontent.com/5165721/279928832-a16b8cb7-5f3c-4364-895f-121e5e08199e.png)
> 
> It probably depends on some other branch. If so, please point me out to it.
> 
> **3. Menu looks different**
> 
> This looks like a side effect of changes. I'll make sure I compiled everything correctly but wanted to highlight it anyway, maybe you see the same.
> 
> ![Screenshot 2023-11-02 at 10 11 22](https://user-images.githubusercontent.com/5165721/279929516-8884ab31-efe1-4efb-808b-1459aafaa9b4.png)

@pawelkmpt 

1. Fixed
2. Removed
3. I tried in both repos while being in the master branch, and it displays the same way.

pawelkmpt commented on packages/cxl-ui/src/components/cxl-light-card.js:
```diff
@@ -14,9 +14,9 @@ export class CXLLightCardElement extends CXLBaseCardElement {
   }
 
-  @property({ type: Number }) progress = 0;
+  @property({ type: String, attribute: 'progress-completed' }) progressCompleted = '0';
 
-  @property({ type: Number }) lessons = 0;
+  @property({ type: String, attribute: 'progress-total' }) progressTotal = '0';
```
Why are these better as string, not number?

pawelkmpt reviewed (changes requested):
_Edit roadmap_ button needs fixing on mobile and tablet screens.

<img width="321" alt="Screenshot 2023-11-02 at 20 27 06" src="https://github.com/conversionxl/aybolit/assets/5165721/1e0fba8b-f976-4b05-be06-e8dc2e87a74b">
<img width="759" alt="Screenshot 2023-11-02 at 20 28 26" src="https://github.com/conversionxl/aybolit/assets/5165721/aaf16a1e-5c41-4d3e-bed8-323d2b525314">

pawelkmpt commented on packages/cxl-ui/scss/cxl-dashboard-header.scss:
```diff
@@ -33,191 +59,98 @@
+    & + .content {
+      margin-top: var(--lumo-space-l);
+      padding-top: var(--lumo-space-l);
+      border-top: 1px solid #d2d5da;
```
Don't we have this color in variable? Or something similar?

pawelkmpt commented on packages/storybook/cxl-ui/cxl-dashboard-header/template.js:
```diff
@@ -12,74 +14,38 @@ export const DashboardHeaderTemplate = (header) => html`
-    .cta2Link=${header.cta2Link}
-    .cta3Link=${header.cta3Link}
-    ?hasRoadmap=${header.hasRoadmap}
+    .showCompletedStats=${header.showCompletedStats}
+    .showContinueSlider=${header.showContinueSlider}
+    .showRoadmap=${header.showRoadmap}
+    .showRoadmapStats=${header.showRoadmapStats}
+    .showRoadmapSlider=${header.showRoadmapSlider}
+    .editRoadmapLinkUrl=${header.editRoadmapLinkUrl}
+    .createRoadmapLinkUrl=${header.createRoadmapLinkUrl}
+    .showMinidegrees=${header.showMinidegrees}
```
Are these suppose to be `camelCase` or `kebab-case` too?

pawelkmpt reviewed (commented):
@HenerHoop one thing caught my eye. Please take a look
